rdrf/rdrf/views/registration_rdrf.py

```diff
@@ -15,6 +17,11 @@ class RdrfRegistrationView(RegistrationView):
     registry_code = None
 
     def get(self, request, *args, **kwargs):
+        try:
+            get_object_or_404(Registry, code=kwargs['registry_code'])
```

Comment:
I think we should test for the registry having the feature 'registration' too:

reg = get_object_or_404(Registry, code=kwargs['registry_code'])
if not reg.has_feature('registration'):
      raise Http404